ctags/main/entry.c

```diff
@@ -366,8 +366,8 @@ extern void openTagFile (void)
 static int replacementTruncate (const char *const name, const long size)
 {
 	char *tempName = NULL;
-	FILE *fp = tempFile ("w", &tempName);
-	fclose (fp);
+	MIO *mio = tempFile ("w", &tempName);
+	fclose (mio);
```

Comment:
@codebrainz a little, but just because of incompatible pointer cast, nothing more